source/uri-options/uri-options.rst

```diff
@@ -210,6 +210,12 @@ pertaining to URI options apply here.
      - required for drivers with connection pools
      - The maximum number of clients or connections able to be created by a pool at a given time. This count includes connections which are currently checked out.
 
+   * - maxPoolConnecting
```

Comment:
I'd suggest that the connection string option matches the `ConnectionPoolOptions` option name, same as with `maxPoolSize`/`minPoolSize`. Going with `maxPoolConnecting` might require replacing all instances of `maxConnecting` in the spec.
Would be great to get @patrickfreed opinion.

Reply:
Using `maxConnecting` in a connection string makes what the option relates to less clear when judging based on the option name. I can rename it to `maxPoolConnecting` in the CMAP spec (I see no problem with renaming per se). Having the "pool" part in the name of a property in CMAP is a useless elaboration as all properties there are those of a pool. However, using the same name everywhere simplifies searching for it. So if I were to pick a single name, I'd pick **`maxPoolConnecting`** because of the connection string argument mentioned above.

Reply:
Renamed to `maxConnecting`. I understand that @BorisDog and @ShaneHarvey agree with this name. @patrickfreed and @JamesKovacs, could you please tell whether you agree with this name for both the URI option and the pool option?

I am asking because we are trying to have this done in the Java driver 4.4., which is released on Thursday (I have only today and tomorrow for the implementation), and we can do that only if the parts visible to users (names of the options in this case) are approved.

Reply:
`maxConnecting` SGTM.

Reply:
`maxConnecting` LGTM. The equivalent `mongos` server parameter that we are emulating is called [ShardingTaskExecutorPoolMaxConnecting](https://docs.mongodb.com/manual/reference/parameters/#mongodb-parameter-param.ShardingTaskExecutorPoolMaxConnecting). I don't see the need to include "pool" explicitly in the name to indicate that it is related to connection pools.